test/e2e/storage/csi_volumes.go

```diff
@@ -102,7 +102,7 @@ var _ = Describe("PMEM Volumes", func() {
 						ClientNodeName: "host-1",
 					},
 				},
-				scManifest: "deploy/kubernetes/pmem-storageclass.yaml",
+				scManifest: "deploy/kubernetes-1.13/pmem-storageclass.yaml",
```

Comment:
A better solution would be to retrieve the Kubernetes version at runtime and then generating the path dynamically. As it stands now, if we ever have to introduce a "deploy/kubernetes-1.14" with a different storage class, we need to remember to update this code.

Reply:
yes, as I stated, another approach would be symlinked dir kubernetes -> kubernetes-1.13 which we have to maintain, but idea there is that we only maintain that single symlink and keep (multiple) occurrences in source as "kubernetes". But as you already started putting exact kubernetes-1.13 in source, I continue that style in this PR. Overall, this multiple directories style is messy of course.